app/actions/ats_client.py

```diff
@@ -7,12 +7,19 @@
 from datetime import datetime, timedelta
 from xml.parsers.expat import ExpatError
 from typing import List, Optional
+from enum import Enum
 
 
 logger = logging.getLogger(__name__)
 
 
 # Pydantic models
+class FileStatus(Enum):
+    PENDING = "pending"
+    IN_PROGRESS = "in_progress"
+    PROCESSED = "processed"
+
```

Comment:
I imagine you moved the enum here to avoid a circular import now that it's used in configurations, but it really doesn't belong to the ats_client since is not related and not used here. I think `configurations.py` would be a better place for it, and you can import it into `handlers.py`.